lib/shoryuken/manager.rb

```diff
@@ -12,7 +12,16 @@ class Manager
 
     def initialize(condvar)
       @count = Shoryuken.options[:concurrency] || 25
-      raise(ArgumentError, "Concurrency value #{@count} is invalid, it needs to be a positive number") unless @count > 0
+      @dispatch_interval = Shoryuken.options[:dispatch_interval] || 1
+
+      unless @count > 0
+        raise(ArgumentError, "Concurrency value #{@count} is invalid, it needs to be a positive number")
+      end
+
+      unless @dispatch_interval > 0
```

Comment:
I think a value of `0` should be allowed here; it's valid to `dispatch(0)` on celluloid if you want to configure a tight loop.